src/alda/AldaClient.java

```diff
@@ -74,7 +74,8 @@ public static void updateAlda()
 
     if (downloadURL == null) {
       throw new UnsuccessfulException(
-        "Alda download link not found for your platform."
+        "Alda download link not found for your platform. Please file an " +
+        "issue at: https://github.com/alda-lang/alda/issues/new"
```

Comment:
This kind of conflicts with alda-lang/alda#293, although this is probably the only time an issue should be filed to the main repository.

Because of that, I'm torn between leaving it as is, and asking people to (mis)file it to alda/alda-core. 

Also, unrelated, but the text to alda-lang/alda#293 regarding the repl is outdated now :smile: